### ISSUE TITLE
Errors in log

### ISSUE THREAD

**Issue by dannybloe** [lendy007/homeassistant-skodaconnect#171]
Ok, I know this may not be helpful but when enabling debug logging for another integration I saw loads of errors like these:

```
2023-01-12 17:40:46.515 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 168, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 316, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 121, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 556, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 597, in _async_write_ha_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 562, in _stringify_state
    if (state := self.state) is None:
  File "/config/custom_components/skodaconnect/sensor.py", line 50, in state
    return self.instrument.state
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 127, in state
    val = super().state
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 58, in state
    if hasattr(self.vehicle, self.attr):
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/vehicle.py", line 1542, in last_connected
    last_connected = datetime.strptime(last_connected_utc,'%Y-%m-%dT%H:%M:%S.%fZ').astimezone().replace(tzinfo=None).replace(microsecond=0)
TypeError: strptime() argument 1 must be str, not None
```
No idea how and why this happens. Rather HA newbe here so please let me know what I can do to help investigate this.

Cheers, 
Danny

**Comment by Farfar** [comment]
"Duplicate" of #157 
Not really a duplicate but the same issue. The API is responding with "UNKNOWN" values instead of the correct format or something sane.
Most values for the API endpoint have been fixed, the last connected is handled differently though so I'm working on a fix.

**Comment by mvzut** [comment]
What's the status of this fix you were mentioning? I have this exact same error in the log as well, it occurs every 5 minutes or so.

**Comment by dvx76** [comment]
Pretty sure I fixed this in skodaconnect/skodaconnect/pull/73

**Comment by Farfar** [comment]
> Pretty sure I fixed this in [skodaconnect/skodaconnect/pull/73](https://github.com/skodaconnect/skodaconnect/pull/73)

Yes you are correct, this specific issue should be fixed. The "duplicates" not however.
